docs/source/dev_environment_setup.md

```diff
@@ -1,5 +1,51 @@
 # Setting up a development environment
 
+## Prerequisites
+
+### Pre-commit hook for Python formatting with black
+
+FlowKit's Python code is formatted with [black](https://black.readthedocs.io/en/stable/) (which is included in the
+FlowKit pipenv environments). There is also a [pre-commit](https://pre-commit.com/) hook which runs black on all Python
+files before each commit.
+
+To install the pre-commit hook run:
+```bash
+$ cd flowmachine && pipenv run pre-commit install
```

Comment:
Don't we want pre-commit to be available/used in the other bits as well?

Reply:
It will be installed and available globally, but the `pre-commit` package is only available in the flowmachine Pipfile, so in order to run the `pre-commit install` command you have to switch to that pipenv environment (since we don't have a global one). :-/

I should add an explanation for this though, good point! 👍

Reply:
Or do you have an idea for a better solution that avoids using a specific pipenv environment?